components/suggest/src/store.rs

```diff
@@ -623,6 +623,31 @@ where
         )
         .unwrap()
     }
+
```

Comment:
Let's add a comment here noting this is really only for development/debugging as it does not care too much about error handling.